### PR TITLE
Privacy Policy

### PR DESCRIPTION
**Pull request by ddollar**
screen shot 2016-05-25 at 3 12 33 pm

### CODE REVIEW

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;yr8W4eK0&#x2F;198-privacy-policy"> privacy policy</a>

File: privacy/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Convox Console collects user names, passwords, and e-mail addresses of users who create accounts using the signup form.  Convox Console also collects and stores administrative credentials for each software deployment platform, or Rack, that you manage with Convox Console.
+
+Some features of Convox Console are only available if you provide information about yourself.
+
+## Collection
+
+Convox Console uses HTTP cookies to remember you, keep you logged in, store your user interface preferences, and track your usage of Convox Console.  Convox Console does not honor Do Not Track request headers.
+
+Convox records requests and others interactions with Convox Console in server logs.  Server logs may record such as which URL your web browser requested, when you made the request, what web browser and kind of device you used to make the request, and other data, such as data typed into a form on Convox Console, passed between your web browser and Convox Console servers.
```
Comment (mattmanning) [inline privacy/index.md]: should this be "other interactions" instead of "others interactions"?

Comment (mattmanning) [inline privacy/index.md]: "Server logs may record such as . . . " feels like it's missing a word after "record"

File: privacy/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Integrations
+
+If you enable Convox Console integrations with third-party services, such as GitHub (<https://github.com>), GitLab (<https://gitlab.com>), and Slack (<https://slack.com>), Convox Console will access and share information about you through those services in order to integrate with them.
+
+For GitHub and GitLab, Convox Console will access information about your user account, as well as your public and private software repositories, in order to build and deploy applications.  Convox Console will share build and other events with Slack.
+
+### Billing
+
+Convox Console uses Stripe (<https://stripe.com>) to collect and process payment card information, such as credit card information, to bill for paid parts of Convox Console.  Stripe collects billing information, stores it, and uses it to charge your payment method on Convox' behalf.
```
Comment (mattmanning) [inline privacy/index.md]: should "Convox' behalf" be "Convox's behalf"?

File: privacy/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Analytics
+
+Convox Console uses Segment (<https://segment.com>) to track usage statistics.  Segment code on Convox Console may use cookies and other tracking methods to collect this information.  Those methods may track your activity over time and across multiple websites.
+
+### Other Privacy Policies
+
+Third-party services may collect, share, and user information about you in other ways, according to their own privacy policies.
```
Comment (mattmanning) [inline privacy/index.md]: should "user information" be "use information"?

**Comment by ddollar** [comment]
For typo stuff like this just make a commit

**Pull request merged**